modules/tinymce/src/core/main/ts/api/util/EventDispatcher.ts

```diff
@@ -7,6 +7,7 @@
 
 import { Fun, Obj } from '@ephox/katamari';
 import Tools from './Tools';
+import { MappedEvent } from './Observable';
```

Comment:
This adds a circular dependency, so that type needs to be moved here instead as it's the lowest level.